### PR TITLE
fix530-531 Revised kitchen count report and labels

### LINKED ISSUE #530: Revise Kitchen Count Report Contents
### Expected Behaviour (from meeting with JFF and JV on 2016-11-30)

- In the summary of quantities, do not show "today's menu and ingredients" except for the main dish
- Remove "meal preparation" column (this will be on the label)
- For each combination of clashing ingredients, place subtotals **above** detail rows
- List only restrictions and ingredients to avoid that are in the client's preferences, and do not list the ingredients that were inferred by SousChef

### PR DESCRIPTION
**Pull request by lamontfr**
### Changes proposed in this pull request:

- delivery/templates/kitchen_count : revise rows and columns layout
- delivery/views : revise kitchen count and labels content
- order/models : revise kitchen count queries and KitchenItem processing

### Status

- [X] WIP

### How to verify this change

- Execute kitchen count steps 1, 2 and 3; view page and click Download labels. Note that the PDF layout of the kitchen count report will be added later as solution for issue #498.

### CODE REVIEW

File: src/delivery/templates/kitchen_count.html [hunk 1]
```diff
@@ -58,26 +58,30 @@ <h1 class="ui header">{% trans "Kitchen Count Report" %}</h1>
...
+            <td class="center aligned">{% firstof obj.lqty "&nbsp;" %}</td>
+            <td>{% firstof obj.client "&nbsp;" %}</td>
         {% endif %}
-        <td class="center aligned">{% firstof obj.rqty "&nbsp;" %}</td>
-        <td class="center aligned">{% firstof obj.lqty "&nbsp;" %}</td>
-        <td>{% firstof obj.client "&nbsp;" %}</td>
-        <td>{% firstof obj.preparation "&nbsp;" %}</td>
         <td>{{obj.rest_ingr}}{% if obj.rest_ingr %} ; {% endif %} <strong>{% firstof obj.rest_item "&nbsp;" %}</strong></td>
       </tr>
     {% endfor %}
```
Comment (lamontfr) [inline src/delivery/templates/kitchen_count.html]: @manumilou I will make the adjustments according to your review. I will also take into accounts the comments on the labels made by Jean-Francois and Jeremie today.

**Comment by manumilou** [comment]
@lamontfr I am a bit lost, sorry. Is it ready or do you have more work to do on this?

**Comment by lamontfr** [comment]
@manumilou I have more work to do. I just changed the status to "Work in progress". Sorry about the confusion.

**Comment by lamontfr** [comment]
Other contributors modified some files that were in my first commit. I decided to close this PR make a fresh one as PR #568.

**Pull request closed without merging**